suse_caasp

```diff
@@ -257,6 +257,31 @@ rpm_verify() {
         fi
 }
 
+# Input logfilename containername command
+docker_exec() {
```

Comment:
thanks for the hint. I'd keep the function generic and append the `entrypoint.sh` to the caller.